README.md

```diff
@@ -1 +1,39 @@
-# api-tools
\ No newline at end of file
+# API Tools
+
+[![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
+
+_A CLI to scrape some really useful UTD data, parse it, and upload it to the Nebula API database for community use._
+
+Part of [Project Nebula](https://about.utdnebula.com)
+
+## Contributing
+
+### Prerequisites
+
+- Golang 1.18.4 (or higher)
+
+### Development
+
+Documentation for the project will be created soon, but for more information please visit our [Discord](https://discord.com/invite/tcpcnfxmeQ)
+
+Clone the repository.
+
+- The `grade-data` directory contains .CSV files of, you guessed it, the UTD grade data!
+- The `main` directory contains the main file that runs the CLI.
+- The `parser` directory contains the files and methods that parses the scraped data.
+- The `scrapers` directory contains the scrapers that scrape the UTD coursebook, the UTD organizations page, and the UTD profiles page.
+- The `uploader` directory contains the uploader that sends the parsed data to the Nebula API database. (Under construction)
+
+#### API Tools (Under construction)
+
+The API Tools use Golang with Gin and the MongoDB Golang Driver.
```

Comment:
`api-tools` doesn't use Gin at all - Gin is the library used for network routing for `nebula-api` :)
Instead, we use ChromeDP for our scrapers and will (eventually) use MongoDB's Golang driver for the uploader.
```suggestion
The API Tools use Golang with ChromeDP and the MongoDB Golang Driver.
```